### PR TITLE
docs: refactor http-logger plugin doc 

### PR DESCRIPTION
**Pull request by hf400159**
### Description
refactor http-logger plugin doc

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/plugins/http-logger.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 title: http-logger
 keywords:
   - APISIX
+  - API 网关
   - Plugin
   - HTTP Logger
-  - http-logger
-description: This document contains information about the Apache APISIX http-logger Plugin.
+description: This document contains information about the Apache APISIX http-logger Plugin, Using this Plugin, you can push APISIX log data to HTTP or HTTPS servers.
```
Comment (SylviaBABY) [inline docs/en/latest/plugins/http-logger.md:8]:
```suggestion
description: This document contains information about the Apache APISIX http-logger Plugin. Using this Plugin, you can push APISIX log data to HTTP or HTTPS servers.
```

File: docs/zh/latest/plugins/http-logger.md [hunk 2]
```diff
@@ -23,32 +31,68 @@ title: http-logger
 
 ## 描述
 
-`http-logger` 是一个插件，可将 Log 数据请求推送到 HTTP / HTTPS 服务器。
-
-这将提供将 Log 数据请求作为 JSON 对象发送到监视工具和其他 HTTP 服务器的功能。
+`http-logger`插件可以将 APISIX 的日志数据推送到 HTTP 或 HTTPS 服务器。该插件提供了将日志数据请求作为 JSON 对象发送到监控工具或者其他 HTTP 服务器的功能。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/http-logger.md:34]:
```suggestion
`http-logger` 插件可以将 APISIX 的日志数据推送到 HTTP 或 HTTPS 服务器。该插件提供了将日志数据请求作为 JSON 对象发送到监控工具或者其他 HTTP 服务器的功能。
```

File: docs/zh/latest/plugins/http-logger.md [hunk 3]
```diff
@@ -23,32 +31,68 @@ title: http-logger
...
-| timeout          | integer | 可选   | 3             | [1,...] | 发送请求后保持连接活动的时间。                   |
-| name             | string  | 可选   | "http logger" |         | 标识 logger 的唯一标识符。                     |
-| include_req_body | boolean | 可选   | false         | [false, true] | 是否包括请求 body。false： 表示不包含请求的 body ； true： 表示包含请求的 body 。 |
-| include_resp_body| boolean | 可选   | false         | [false, true] | 是否包括响应体。包含响应体，当为`true`。 |
-| include_resp_body_expr | array  | 可选    |           |         | 是否采集响体，基于 [lua-resty-expr](https://github.com/api7/lua-resty-expr)。 该选项需要开启 `include_resp_body`|
-| concat_method    | string  | 可选   | "json"        | ["json", "new_line"] | 枚举类型： `json`、`new_line`。**json**: 对所有待发日志使用 `json.encode` 编码。**new_line**: 对每一条待发日志单独使用 `json.encode` 编码并使用 "\n" 连接起来。 |
-| ssl_verify       | boolean | optional    | false          | [false, true] | 是否验证证书。 |
+| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头。则表明获取 [APISIX 变量](../../../en/latest/apisix-variable.md) 或 [NGINX 内置变量](http://nginx.org/en/docs/varindex.html)。 |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/http-logger.md:60]:
```suggestion
| log_format       | object  | 否   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头。则表明获取 [APISIX 变量](../../../en/latest/apisix-variable.md) 或 [NGINX 内置变量](http://nginx.org/en/docs/varindex.html)。 |
```

File: docs/zh/latest/plugins/http-logger.md [hunk 4]
```diff
@@ -65,49 +109,23 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
-```
-
-## 插件元数据设置
-
-| 名称             | 类型    | 必选项 | 默认值        | 有效值  | 描述                                             |
-| ---------------- | ------- | ------ | ------------- | ------- | ------------------------------------------------ |
-| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md) 或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。特别的，**该设置是全局生效的**，意味着指定 log_format 后，将对所有绑定 http-logger 的 Route 或 Service 生效。 |
-
-### 设置日志格式示例
+[mockbin](http://mockbin.org/bin/create)  服务器用于模拟 HTTP 服务器，以方便查看 APISIX 生成的日志。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/http-logger.md:112]:
```suggestion
[mockbin](http://mockbin.org/bin/create) 服务器用于模拟 HTTP 服务器，以方便查看 APISIX 生成的日志。
```

**Review by SylviaBABY — APPROVED** [review]
LGTM

**Review by SylviaBABY — APPROVED** [review]

File: docs/zh/latest/plugins/http-logger.md [hunk 5]
```diff
@@ -23,32 +31,68 @@ title: http-logger
...
-| timeout          | integer | 可选   | 3             | [1,...] | 发送请求后保持连接活动的时间。                   |
-| name             | string  | 可选   | "http logger" |         | 标识 logger 的唯一标识符。                     |
-| include_req_body | boolean | 可选   | false         | [false, true] | 是否包括请求 body。false： 表示不包含请求的 body ； true： 表示包含请求的 body 。 |
-| include_resp_body| boolean | 可选   | false         | [false, true] | 是否包括响应体。包含响应体，当为`true`。 |
-| include_resp_body_expr | array  | 可选    |           |         | 是否采集响体，基于 [lua-resty-expr](https://github.com/api7/lua-resty-expr)。 该选项需要开启 `include_resp_body`|
-| concat_method    | string  | 可选   | "json"        | ["json", "new_line"] | 枚举类型： `json`、`new_line`。**json**: 对所有待发日志使用 `json.encode` 编码。**new_line**: 对每一条待发日志单独使用 `json.encode` 编码并使用 "\n" 连接起来。 |
-| ssl_verify       | boolean | optional    | false          | [false, true] | 是否验证证书。 |
+| log_format       | object  | 否    | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头。则表明获取 [APISIX 变量](../../../en/latest/apisix-variable.md) 或 [NGINX 内置变量](http://nginx.org/en/docs/varindex.html)。 |
```
Comment (juzhiyuan) [inline docs/zh/latest/plugins/http-logger.md:60]: How about using documentation's URL?

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**